docs/API_reference.md

```diff
@@ -562,6 +564,36 @@ There are currently two supported output formats: 'json' and 'dataframe'.
 The 'json' format simply returns the raw response, without page information and with multiple pages combined into a single data structure.  
 The 'dataframe' format transforms that data structure into a pandas DataFrame, which contains subparts in json, that can be split further by the user application.
 
+#### Plenty BI related data <a name='get-bi-section'></a>
+
+##### plenty_api_get_contacts <a name='get-bi'></a>
+
+Get list of raw data files and (optionally) download raw data.
+
+[*Optional parameter*]:
+**dataName**:  restricts the search results to raw data files e.g. orders,orderItems.
+Multible raw data files can be separated by comma.
+
+**processStatus**:  Current process status, the status is only changed by internal processing. If this filter is used, the filter dataNames may only contain one value.
+
+**createdAtTimestamp**: UNIX-Timestamp from when daily generated raw data are to be filtered. This filter cannot be combined with the following filters: dataNames, processStatus
```

Comment:
As this is a library we can enforce this rule before the request is made within `sanity_check_parameter`:
```python3
@@ -283,6 +284,15 @@ def sanity_check_parameter(domain: str,
             logging.info(f"Invalid refine argument key removed: {invalid_keys}")
             for invalid_key in invalid_keys:
                 refine.pop(invalid_key, None)
+        if domain == 'bi_raw':
+            if (
+                'createdAtTimestamp' in refine and
+                any(x in refine for x in ['dataNames', 'processStatus'])
+            ):
+                raise RuntimeError(
+                    "Invalid combination of refine arguments. 'createdAtTimestamp'"
+                    "may not be combined with 'dataNames' or 'processStatus'"
+                )
```

Reply:
I can't make sense of the API documentation: "cannot be combined with the following filters: dataNames, processStatus", but that is not true at all. 

I have an cronjob, which downloads several raw data on a daily basis. I just keep track of the Timestamp and use the last timestamp and dataName as refine params. Works flawless.

Also the Plenty API doc says: "dataName", but later they talk about "dataName**s**". 

In fact 
`items = plenty.plenty_api_get_bi_raw_files(refine={'dataName':'orderItemAmounts,orderItems'})`
doesn't work, it fetches all available data (e.g. systemCloudUsage...) 

I asked the question in the forum. I would put that on hold until I have a definitive answer on this.